avaframe/ana1Tests/energyLineTest.py

```diff
@@ -228,13 +223,17 @@ def generateCom1DFAEnergyPlot(avalancheDir, energyLineTestCfg, avaProfileMass, d
     return errorEnergyTest
 
 
-def getRunOutAngle(avaProfileMass):
+def getRunOutAngle(avaProfileMass, indStart=0, indEnd=-1):
     """Compute the center of mass runout angle
 
     Parameters
     -----------
     avaProfileMass: dict
         particle mass average properties
+    indStart: int
+        index of the start of the mass averaged pass (to discard the top extension). 0 by default
```

Comment:
0 means that you take the first point as start. Whether the first point is in the extension depends on your application.
I made the function like this so it can be used in other cases

Reply:
I see, thanks! So I would just suggest the following:
index of the start of the mass averaged path (to enable e.g. to discard the top extension). 0 by default - so full mass averaged path from top

and it is path not pass I think